integration/src/test/java/org/kie/hacep/RemoteKieSessionImplTest.java

```diff
@@ -64,15 +67,19 @@ public void getFactCountTest() {
         kafkaServerTest.insertBatchStockTicketEvent(7,
                                                     topicsConfig,
                                                     RemoteKieSession.class);
-        try (RemoteKieSessionImpl client = new RemoteKieSessionImpl(Config.getProducerConfig("getFactCountTest"),
-                                                                    topicsConfig)) {
+        RemoteKieSessionImpl client = new RemoteKieSessionImpl(Config.getProducerConfig("getFactCountTest"),
+                                                               topicsConfig);
+        try {
             client.fireUntilHalt();
             client.listen();
             CompletableFuture<Long> factCountFuture = client.getFactCount();
-            Long factCount = factCountFuture.get(20, TimeUnit.SECONDS);
+            Long factCount = factCountFuture.get(20,
+                                                 TimeUnit.SECONDS);
             assertTrue(factCount == 7);
-        } catch (Throwable e) {
-            e.printStackTrace();
+        }catch (Exception e) {
+            logger.error(e.getMessage(), e);
```

Comment:
You are not propagating the exception so the test will not fail. Do not log it here, just throw it or even better declare `throws` in method name and junit will take care of it. In this way you can get rid of `LoggerFactory` class too

Reply:
+1

Reply:
Ok